Packs/EDL/Integrations/EDL/README.md

```diff
@@ -1,13 +1,22 @@
 This integration provides External Dynamic List (EDL) as a service for the system indicators (Outbound feed). The feed content can then be sent to 3rd parties for improving their security alignment.
 
+## PAN-OS EDL Management to PAN-OS EDL Service migration steps
+This integration hosts the EDL on the Cortex XSOAR server. Please follow these steps to migrate your EDLs.
+1. Create IOCs in Cortex XSOAR from the EDLs data, which was previously hosted on an external web server. To do so automatically, extract your EDL as a text file from the web server, upload it as a file to the war-room and use the `ExtractIndicatorsFromTextFile` automation. e.g, `!ExtractIndicatorsFromTextFile entryID=<entry_id>` 
+2. Go to the `Indicators` and [filter](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/5-5/cortex-xsoar-admin/manage-indicators/understand-indicators/indicators-page.html#idf15421df-a0e0-41fb-b8d4-07d0f610bbec) to find all of the Indicators you extracted from the text file.
+3. If needed, batch select the indicators and add a tag to indicators you which to host as a specific EDL. Use this tag in the `Indicator Query` integration parameter when configuring the integration. 
+4. Maintain your indicators by understanding the (expiration)[https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/manage-indicators/understand-indicators/indicator-expiration.html] policy and the [exclusion](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/manage-indicators/understand-indicators/exclusion-list.html) list.
```

Comment:
not sure about this comment here - i think it can be removed

Reply:
let's wait for dean/alex. I think it might be tricky as IPS on the EDL live forever, ours live 14 days by default.

Reply:
when you say 'live' - you mean the indicator, not the EDL.
i went over the pack and it doesn't seem like we even mention expiration anywhere (we don't provide a default query, for example).
the EDL can be configured to whatever query - meaning the user doesn't have to take expiration into consideration.

Reply:
You convinced me. I'm thinking that when migrating from a constant hard coded list to q query of a dynamic DB, we need to point them to more KB so they will feel in control. probably does not belong here.